dbt-cta/action_network/models/1_cta_full_refresh/core_fields_counties_base.sql

```diff
@@ -13,6 +13,7 @@
     partitions = partitions_to_replace,
     unique_key = "_airbyte_raw_id"
```

Comment:
Is this still ok as the `unique_key` or do we want it to be `id` like the `incremental` configs?

Reply:
(same with the other 2 `full_refresh`)

Reply:
Oh yeah! good catch - these do have primary keys (or at least fields called `id`) but we aren't syncing them incrementally in Airbyte because they don't have cursor fields. So yep I'll make these unique_keys be `id` as well (though since these are full refresh I think the unique_key doesn't actually impact dbt's behavior)

Reply:
https://github.com/community-tech-alliance/dbt-cta/pull/356/commits/4f3a057a85b16b4cfabe24d507e30fb8c45cd865